vhdbuilder/packer/install-dependencies.sh

```diff
@@ -77,12 +77,19 @@ else
   done
 fi
 
+mkdir -p /etc/systemd/system/chrony.service.d
+tee -a /etc/systemd/system/chrony.service.d/10-chrony-restarts.conf > /dev/null <<'EOF'
```

Comment:
nit: can use `cat` without the need to redirect output to the null device

Reply:
Done. Cat is far nicer!